script/deploy

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# Blue-green deployment script. Usage:
+#
+#   ./script/deploy <appname>
+#
+# Based on
+#   http://www.cloudfoundry.rocks/blue-green-deployment-with-cloudfoundry/
+#   http://docs.pivotal.io/pivotalcf/devguide/deploy-apps/blue-green.html
+#   https://github.com/dlapiduz/step-cloud-foundry-deploy/blob/master/run.sh
+
+set -e
+set -x
+
+PRIMARY=$1
```

Comment:
I wouldn't call it primary, secondary. That can imply that one is better than the other. We're replacing the primary with the secondary sounds like we're replacing it with something inferior.